modules/dynamic-provisioning-aws-definition.adoc

```diff
@@ -13,7 +13,7 @@ kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
   name: <storage-class-name> <1>
-provisioner: kubernetes.io/aws-ebs
+provisioner: ebs.csi.aws.com
```

Comment:
CSI drivers can have different `parameters` than corresponding in-tree volume plugins.
For AWS EBS, 4.14 understands these options: https://github.com/openshift/aws-ebs-csi-driver/blob/release-4.14/docs/parameters.md (replace `4.14` in the URL with a different versions, 4.15 got a few new ones)

It seems that our documentation is still valid for AWS EBS. The CSI driver has more options that we don't document yet. And the question is if we should, because it's a huge effort - we would need to test the options first.

Reply:
@jsafrane 
How about just listing the OCP pre-defined storageclass content (like https://github.com/openshift/aws-ebs-csi-driver-operator/blob/master/assets/storageclass_gp3.yaml) in this ["dynamic-provisioning" section](https://71398--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/dynamic-provisioning) and documenting more parameters when introducing the CSI Driver section?

Reply:
That would work for me